staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation/validation.go

```diff
@@ -557,11 +569,11 @@ func hasValidConversionReviewVersionOrEmpty(versions []string) bool {
 }
 
 // ValidateCustomResourceConversion statically validates
-func ValidateCustomResourceConversion(conversion *apiextensions.CustomResourceConversion, fldPath *field.Path) field.ErrorList {
-	return validateCustomResourceConversion(conversion, true, fldPath)
+func ValidateCustomResourceConversion(conversion *apiextensions.CustomResourceConversion, fldPath *field.Path, opts validationOptions) field.ErrorList {
```

Comment:
Adding an unexported type to the `ValidateCustomResourceConversion` signature makes this function unable to be called by anyone outside the package.

Options:
1. delete the function
2. remove the opts from the signature and pass `validationOptions{}` to `validateCustomResourceConversion`
3. export `validationOptions` type

Since I don't see any callers of this function in-tree or out-of-tree, I'd favor option 1, but would also accept 2 if anyone felt strongly

Reply:
ack, also in favor of option 1. Removed the function.